code/obj/item/clothing/shoes.dm

```diff
@@ -382,6 +382,25 @@
 		..()
 		setProperty("meleeprot", 1)
 
+/obj/item/clothing/shoes/clog
+	name = "Dutch clogs"
+	desc = "A pair of wooden clogs that people from Holland wear."
+	icon_state = "clog"
+	permeability_coefficient = 0.30
+	kick_bonus = 2 //kicking someone with wooden shoes has gotta hurt
+	step_sound = "step_wood"
+	step_priority = STEP_PRIORITY_LOW
+	setupProperties()
+		..()
+		setProperty("meleeprot", 1)
+
+	get_desc(var/dist, var/mob/user)
+		if (user.traitHolder.hasTrait("dutch"))
+			. += "Remembers you of the homeland."
```

Comment:
I'm not a native speaker of English but to me this looks like it should be "Reminds" and not "Remembers". Ignore if I'm wrong or of the effect is intended.

Reply:
"Reminds" is the proper English.